### DISCUSSION TITLE
Is graphql-http supported?

### DISCUSSION THREAD

**Discussion by oddball** [dotansimha/envelop#1770] [General]
Hi!

Can you use envelop with https://www.npmjs.com/package/graphql-http ? 

If you can, is there an example somewhere? I could not find it here https://the-guild.dev/graphql/envelop/docs/integrations

**Comment by n1ru4l** [comment]
Our recommended way of using envelop is through Yoga (https://github.com/dotansimha/graphql-yoga).

However, envelop should also work with `graphql-http`. We have a graphql-ws example which has a very similar API: https://github.com/n1ru4l/envelop/tree/main/examples/graphql-ws

@enisdenjo It might make sense to also add a graphql-http example in this repo?

**Reply by enisdenjo** [reply]
Yep, I've added myself a task. Will handle it. 👌
